test/vintage_net_bridge_test.exs

```diff
@@ -92,5 +92,69 @@ defmodule VintageNetBridgeTest do
     assert output == VintageNetBridge.to_raw_config("br0", input, Utils.default_opts())
   end
 
+  test "bridge with static ip" do
+    input = %{
+      type: VintageNetBridge,
+      ipv4: %{
+        method: :static,
+        address: "169.254.169.254",
+        prefix_length: 16
+      },
+      vintage_net_bridge: %{
+        interfaces: ["eth0", "tap0"]
+      },
+      hostname: "unit_test"
+    }
+
+    ipv4_address_to_tuple = fn address ->
+      address
+      |> String.split(".")
+      |> Enum.map(&String.to_integer/1)
+      |> List.to_tuple()
+    end
+
+    output = %RawConfig{
+      ifname: "br0",
+      type: VintageNetBridge,
+      source_config: update_in(input[:ipv4][:address], &ipv4_address_to_tuple.(&1)),
```

Comment:
Converting `input` config's `address string` to `integer tuple` is needed to compare.